### PR TITLE
docs update to topics/voice playFile example

### PR DESCRIPTION
**Pull request by Dev-Yukine**
**Please describe the changes this PR makes and why it should be merged:**
in the docs under topics/voice there is said that you need to give .playFile() an absolut path what is wrong since my own bot use dynamic paths with .playFile() and it works fine so i added a dynamic path example aswell as update the ReadableStream example

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: docs/topics/voice.md [hunk 1]
```diff
@@ -46,11 +46,17 @@ have obtained a voice connection, we can start streaming audio to it. The follow
 file:
 
 **Playing a file:**
+
 ```js
-// To play a file, we need to give an absolute path to it
+//use an absolute path
```
Comment (iCrawl) [inline docs/topics/voice.md]: Space + Uppercase first letter.

File: docs/topics/voice.md [hunk 2]
```diff
@@ -46,11 +46,17 @@ have obtained a voice connection, we can start streaming audio to it. The follow
...
 const dispatcher = connection.playFile('C:/Users/Discord/Desktop/myfile.mp3');
 ```
 
+```js
+//or an dynamic path
```
Comment (iCrawl) [inline docs/topics/voice.md]: Space + Uppercase first letter.

File: docs/topics/voice.md [hunk 3]
```diff
@@ -84,8 +90,7 @@ this can also be used:
 ```js
 connection.playStream(myReadableStream);
 
-// If you don't want to use absolute paths, you can use
-// fs.createReadStream to circumvent it
+//you can use fs.createReadStream to create an ReadableStream
```
Comment (iCrawl) [inline docs/topics/voice.md]: Space + Uppercase first letter.

File: docs/topics/voice.md [hunk 4]
```diff
@@ -94,6 +99,7 @@ connection.playStream(stream);
 
 It's important to note that creating a readable stream to a file is less efficient than simply using `connection.playFile()`.
 
+
```
Comment (iCrawl) [inline docs/topics/voice.md]: Why?

Comment (Dev-Yukine) [inline docs/topics/voice.md]: Oh i didn't even notice i did that will remove it sorry!

**Pull request merged**